discoverer/src/main/java/eu/seaclouds/platform/discoverer/core/OfferingManager.java

```diff
@@ -283,5 +286,48 @@ public String getSingleOffering() {
 
         return ret;
     }
+
+    public void initializeFromRemote(String remoteInitializationPath) {
+        try {
```

Comment:
You are not closing resources here. You could try using this: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html